view/rooms/index.pug

```diff
@@ -77,5 +77,6 @@ html
     var trackingId = '#{trackingId}';
     var user = '#{user}';
     var userSession = '#{userSession}';
+    var useNewNamespaceFormat = #{useNewNamespaceFormat};
```

Comment:
should this be surrounded in ' ' like the others?

Reply:
This is a boolean, whereas the others are strings.